compiler/circle-partitioner/src/PartitionRead.cpp

```diff
@@ -69,13 +92,33 @@ luci::PartitionTable parse_table(const crew::Sections &sections)
       for (auto &item : items)
       {
         if (item.first == _key_underscore)
-          table.default_group = item.second;
+        {
+          if (table.comply == luci::PartitionTable::COMPLY::OPCODE)
+            table.default_group = item.second;
+        }
```

Comment:
This change is to prevent setting default group from wrong section (not the section of comply)